### ISSUE TITLE
is there any method to finetune this clip model on other cloth or  E-commerce datasets？i want to finetune in my own datasets which include pictures and texts.

### ISSUE THREAD

**Issue by Originlightwkp** [patrickjohncyh/fashion-clip#17]
i want to finetune in my own datasets which include pictures and texts. Thank you!

**Comment by vinid** [comment]
Hello!

Yes! You can load it in HF and get scores for input pairs. You should be able to useadd a Dataset object on top of the following+the loss to tune it:

```python
from PIL import Image
import requests
from transformers import CLIPProcessor, CLIPModel

model = CLIPModel.from_pretrained("patrickjohncyh/fashion-clip")
processor = CLIPProcessor.from_pretrained("patrickjohncyh/fashion-clip")

image = Image.open("images/image1.jpg")

inputs = processor(text=["a photo of a red shoe", "a photo of a black shoe"],
                   images=image, return_tensors="pt", padding=True)

outputs = model(**inputs)
logits_per_image = outputs.logits_per_image  # this is the image-text similarity score
probs = logits_per_image.softmax(dim=1)  
print(probs)
image.resize((224, 224))
```
